staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -168,6 +177,9 @@ func patchResource(
 	// and is given the currently persisted object as input.
 	applyPatch := func(_ request.Context, _, currentObject runtime.Object) (runtime.Object, error) {
 		// Make sure we actually have a persisted currentObject
+		if trace != nil {
```

Comment:
It seems more reasonable to require a non nil trace.

Reply:
Done.